src/MoBi.Presentation/Mappers/IndividualParameterToParameterDTOMapper.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+using MoBi.Presentation.DTO;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Formulas;
+using OSPSuite.Core.Domain.Mappers;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility.Extensions;
+
+namespace MoBi.Presentation.Mappers
+{
+   public interface IIndividualParameterToParameterDTOMapper
+   {
+      ParameterDTO MapFrom(IndividualBuildingBlock individualBuildingBlock, IndividualParameter individualParameter);
+   }
+   
+   public class IndividualParameterToParameterDTOMapper : IIndividualParameterToParameterDTOMapper
+   {
+      private readonly IParameterToParameterDTOMapper _parameterToParameterDTOMapper;
+      private readonly IParameterValueToParameterMapper _parameterValueToParameterMapper;
+      private readonly IFormulaFactory _formulaFactory;
+      private readonly ICloneManagerForBuildingBlock _cloneManager;
+
+      public IndividualParameterToParameterDTOMapper(IParameterToParameterDTOMapper parameterToParameterDTOMapper, IParameterValueToParameterMapper parameterValueToParameterMapper, IFormulaFactory formulaFactory, ICloneManagerForBuildingBlock cloneManager)
+      {
+         _parameterToParameterDTOMapper = parameterToParameterDTOMapper;
+         _parameterValueToParameterMapper = parameterValueToParameterMapper;
+         _formulaFactory = formulaFactory;
+         _cloneManager = cloneManager;
+      }
+      
+      public ParameterDTO MapFrom(IndividualBuildingBlock individualBuildingBlock, IndividualParameter individualParameter)
+      {
+         var parameterDTO = _parameterToParameterDTOMapper.MapFrom(parameterForIndividualParameter(individualBuildingBlock, individualParameter));
```

Comment:
This is a bit hard to read for me. Personal opinion would be to create a local parameter and then called MapFrom